src/messages/src_authority.rs

```diff
@@ -41,7 +41,7 @@ pub enum SrcAuthority {
     },
     /// Authority of a whole section.
     Section {
-        /// Name in the source section.
+        /// Name of the source section.
```

Comment:
I think `in` is more accurate. `name of a section` will be just prefix tailed with all zero.